server/mergin/sync/public_api_controller.py

```diff
@@ -535,19 +556,19 @@ def get_projects_by_uuids(uuids):  # noqa: E501
     if len(proj_ids) > 10:
         abort(400, "Too many projects")
 
-    user_ids = []
-    ws_ids = []
     projects = (
         projects_query(ProjectPermissions.Read, as_admin=False)
         .filter(Project.id.in_(proj_ids))
         .all()
     )
-    for p in projects:
-        # FIXME
-        # user_ids.extend(p.access.owners + p.access.writers + p.access.readers)
-        ws_ids.append(p.workspace_id)
+    ws_ids = [p.workspace_id for p in projects]
```

Comment:
Do we want to have unique workspace ids?

Reply:
yea, let's do it, not that it would affect much